figure_notebooks/supp_analyze_journalist_data/journalist_contingency_tab.md

```diff
@@ -0,0 +1,1114 @@
+<tr class="odd">
+<td align="left">CelticEnglish</td>
+<td align="right">0.0100874</td>
+<td align="right">0.0235373</td>
+<td align="right">0.3705447</td>
+<td align="right">0.0867518</td>
+<td align="right">0.3665098</td>
+<td align="right">0.0100874</td>
+<td align="right">0.0215198</td>
+<td align="right">0.0295898</td>
+<td align="right">0.0564896</td>
+<td align="right">0.0248823</td>
+</tr>
+<tr class="even">
+<td align="left">EastAsian</td>
+<td align="right">0.0000000</td>
+<td align="right">0.1000000</td>
+<td align="right">0.4583333</td>
+<td align="right">0.1166667</td>
+<td align="right">0.1666667</td>
+<td align="right">0.0000000</td>
+<td align="right">0.0083333</td>
+<td align="right">0.0666667</td>
+<td align="right">0.0083333</td>
+<td align="right">0.0750000</td>
+</tr>
+<tr class="odd">
+<td align="left">European</td>
+<td align="right">0.0208791</td>
+<td align="right">0.0296703</td>
+<td align="right">0.4109890</td>
+<td align="right">0.1076923</td>
+<td align="right">0.3000000</td>
+<td align="right">0.0120879</td>
+<td align="right">0.0230769</td>
+<td align="right">0.0186813</td>
+<td align="right">0.0307692</td>
+<td align="right">0.0461538</td>
+</tr>
+</tbody>
+</table>
+
+## Make Plots
+
+``` r
+#### plot the overview of gender
+journo_gender_quote_df = rbind(female_j_prop_df, male_j_prop_df)
+
+journo_gender_quote_gg = 
+    ggplot(journo_gender_quote_df, 
+      aes(x=as.numeric(year), y=mean,
+                          ymin=bottom_CI, ymax=top_CI,
+                          fill=corpus)) +
+    geom_point() + geom_ribbon(alpha=0.5) + geom_line(alpha=0.5) + theme_bw() + 
+    xlab("Year of Article") + ylab("Male Percentage") +
+    ggtitle("Est. Proportion Male Quotes by Journalist Est. Gender") + 
+    ylim(c(0, 1)) +
+    geom_hline(yintercept=0.5, color="red") +
+    scale_fill_manual(values=c("#E69F00", "#56B4E9")) +
+    theme(legend.position="bottom")
+
+ggsave(file.path(proj_dir,  "/figure_notebooks/supp_analyze_journalist_data/tmp_files/journo_gender_quote_gg.pdf"),
+       journo_gender_quote_gg, width = 6, height = 5, units = "in", device = "pdf")
+
+
+journo_gender_quote_gg
+```
+
+![](journalist_contingency_tab_files/figure-markdown_github/make_citation_gg-1.png)
+
+## format supp. figure XX
+
+``` r
+journo_gender_gg = image_read_pdf(file.path(proj_dir,
+                                  "/figure_notebooks/supp_analyze_journalist_data/tmp_files/journo_gender_gg.pdf"))
+journo_gender_gg = image_annotate(journo_gender_gg, "a", size = 30)
+
+
+journo_gender_quote_gg = image_read_pdf(file.path(proj_dir,
+                                  "/figure_notebooks/supp_analyze_journalist_data/tmp_files/journo_gender_quote_gg.pdf"))
+journo_gender_quote_gg = image_annotate(journo_gender_quote_gg, "b", size = 30)
+
+
+top_image <- image_append(image_scale(c(journo_gender_gg,
+                                           journo_gender_quote_gg),3000), stack = FALSE)
+
+print(top_image)
+```
+
+    ## # A tibble: 1 x 7
```

Comment:
Is panel B the estimated number of male quotes reported by journalist gender? I'm not sure I follow this panel.

Reply:
Yes, exactly! I'll add some more context in the description.

Reply:
added here: https://github.com/greenelab/nature_news_disparities/pull/33/commits/c10d89d645984b96bad146e49b99d081bd9c8dba